Dockerfile

```
@@ -9,22 +9,25 @@ RUN apt-get update \
 && apt-get install gcc -y \
 && apt-get clean
 
-# Set the working directory
-ENV APP_HOME /app
-WORKDIR $APP_HOME
+# Install pipenv
+RUN python -m pip install --upgrade pip \
```

Hadolint (2.12.0)

[error] Expected end of double quoted string. Fix any mentioned problems and try again.

(SC1072)